### PR TITLE
feat: Tree support draggable handler

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Tree `draggable` add handler for tips.       |
| 🇨🇳 Chinese |   Tree `draggable` 添加拖拽图标以提示。       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/tree/Tree.tsx [hunk 1]
```diff
@@ -183,6 +184,7 @@ const Tree = React.forwardRef<RcTree, TreeProps>((props, ref) => {
       switcherIcon={(nodeProps: AntTreeNodeProps) =>
         renderSwitcherIcon(prefixCls, switcherIcon, showLine, nodeProps)
       }
+      draggableIcon={<HolderOutlined />}
```
Comment (afc163) [inline components/tree/Tree.tsx:187]:
```jsx
draggable={{
  icon: React.ReactNode,  // 图标可配
  showHandler: boolean,   // 允许隐藏图标，避免惊吓
}}
```

Comment (zombieJ) [reply components/tree/Tree.tsx:187]: 简单一点如何：
```ts
type DraggableFn = (node: DataNode) => boolean;

draggable?: DraggableFn | boolean | {
  icon?: React.ReactNode | false;
  draggable?: DraggableFn;
};
```

Comment (afc163) [reply components/tree/Tree.tsx:187]: type DraggableFn = (node: DataNode) => boolean; // 这个 boolean 也需要支持 {}

Comment (afc163) [reply components/tree/Tree.tsx:187]: nodeDraggable 是啥意思？

**Comment by zombieJ** [comment]
再看看，我准备升其他几个 rc 版本依赖了

**Comment by afc163** [comment]
图标上是不是应该加个拖拽手势。

**Comment by zombieJ** [comment]
主要都可以拖，这里加了会让人以为只能拖 Handler 我特地不加的。

**Pull request merged**